SIG Service Catalog's charter

carolynvs opened the pull request:
Draft of the SIG Service Catalog charter. Announcement on SIG mailing list to follow.

The first commit contains the current working rules that the chairs have been following so far. Proposed changes are included in separate commits.

/cc @arschles @pmorie @vaikas-google @duglin

carolynvs commented:
/committee steering
/hold

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the OSBAPI specification.
```
Perhaps make "OSBAPI specification" a link to the spec's master version?

duglin replied:
did you want to address this one?

carolynvs replied:
heh, I added the link on line 14, not 12. Fixing

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes leadership teams.
+  - Chairs MAY decide to step down at anytime and MAY propose a replacement, ideally from existing
+    maintainers. Use lazy consensus amongst chairs with fallback on majority vote to accept proposal.
```
This is inconsistent with line 28 where is unanimous for new maintainers. Not sure why this is a lower bar. Let's keep it at unanimous since that what it is today.

duglin replied:
Thinking more about this - I think we should just separate this into two parts - steeping down and adding a new one. Stepping down can be at any time w/o a vote - we can't force someone to stay.  Then the replacement side is just following whatever rule we define for adding a new one.

carolynvs replied:
Agreed it doesn't need to be two separate rules. I'll follow up in a sec to make that consistent.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes leadership teams.
+  - Chairs MAY decide to step down at anytime and MAY propose a replacement, ideally from existing
+    maintainers. Use lazy consensus amongst chairs with fallback on majority vote to accept proposal.
+    This SHOULD be supported by a majority of SIG members.
```
Not sure what this mean. Formal vote? We don't do this today. I would remove line 38.

arschles replied:
agree on removing

carolynvs replied:
That was from the template, but I agree it's unclear. I'll remove in a sec.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+  - Chairs MAY decide to step down at anytime and MAY propose a replacement, ideally from existing
+    maintainers. Use lazy consensus amongst chairs with fallback on majority vote to accept proposal.
+    This SHOULD be supported by a majority of SIG members.
+  - Chairs MAY select additional chairs through a super-majority vote amongst chairs.
```
This is inconsistent with the unanimous stuff from before and the "majority" rule on line 37. I would just combine all of this char stuff into one line that says something like "chairs (new, selecting a replacement, or removal) are nominated by an existing chair and approved by a unanimous chair vote" or something like that.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+    maintainers. Use lazy consensus amongst chairs with fallback on majority vote to accept proposal.
+    This SHOULD be supported by a majority of SIG members.
+  - Chairs MAY select additional chairs through a super-majority vote amongst chairs.
+    This SHOULD be supported by a majority of SIG members.
```
Remove line 40 - or at least until we define what the heck it means.

arschles replied:
+1 on removing

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+    This SHOULD be supported by a majority of SIG members.
+  - Chairs MUST remain active in the role and MAY removed from the position
+    if they are unresponsive for > 3 months or are not proactively working
+    with other chairs to fulfill responsibilities. Use lazy consensus amongst chairs with
```
again, let's be consistent w.r.t. voting rules.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+  - Chairs have access to manage this repository.
+- [Helm Repository](https://svc-catalog-charts.storage.googleapis.com)
+  - Charts are manually published after each release.
+  - Managed by Vic Iglesias (Google).
```
please add his email or some kind of contact info

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+    in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes leadership teams.
+  - Chairs MAY decide to step down at anytime and MAY propose a replacement, ideally from existing
```
we're not consistent on our use of RFC2119 keywords - I would recommend that we don't bother with it and just lowercase uses like this.

carolynvs commented:
FYI, the proposals commits (such as the security contact and the chair step down rules) were copied nearly verbatim from the governance template. I would prefer to edit our other charter draft language existing towards that instead of diverging further from the steering committee's guidance.

pmorie commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+    with other chairs to fulfill responsibilities. Use lazy consensus amongst chairs with
+    fallback on majority vote.
+
+- Emeritus Chairs ([Inspired by the Helm Project](http://technosophos.com/2018/01/11/introducing-helm-emeritus-core-maintainers.html))
```
'Chair' is a special status in my opinion - it comes with a significant amount of responsibility and obligation to perform those responsibilities. I'm in favor of having the emeritus role to give people a healthy way to bow out of the project if they can't/don't want to perform those duties any longer.

arschles replied:
Also in favor. Additional information/justification for emeritus:

http://technosophos.com/2018/01/11/introducing-helm-emeritus-core-maintainers.html

pmorie commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,128 @@
+- While everyone is encouraged to contribute to the discussion of
+  any topic, ultimately any changes made to the codebase must be
+  approved by the maintainers.
+- Pull requests are required to have at least 2 maintainers approve them using the `LGTM 1` and `LGTM 2` labels.
```
While this text reflects how we currently operate, we've also discussed moving to prow, which comes with a different process. One thing this charter doesn't currently deal with is the process for changing the charter itself. I wouldn't want to tie ourselves in a knot because we don't know how to change our own charter - let's make sure there's language for this before this PR is merged.

carolynvs replied:
I've updated the charter with how to change it in the future.

carolynvs commented:
I have incorporated the most recent review feedback, made the decision making process for chairs consistent, and included how to modify the charter.

arschles commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,131 @@
+  - Canary builds are published on pushes to master.
+  - Release builds (and latest) are published on tags.
+  - Files hosted on Azure blob storage.
+  - Azure account managed by Carolyn Van Slyck (Microsoft) and Aaron Schelsinger (Microsoft).
```
s/Schelsinger/Schlesinger

carolynvs replied:
zomg I misspelled my boss's name 😊  _hides under a rock_

arschles replied:
lol no worries 😆

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,131 @@
+    This process is used for all decisions, such as changing chairs/maintainers or modifying this charter.
+  - Chairs may nominate a new chair at any time, to be accepted via lazy two-thirds resolution amongst the chairs.
+  - Chairs may decide to step down at any time. Before stepping down, the chair
+    may propose and vote on their replacement via lazy two-thirds resolution amongst the chairs.
```
This implies there is a fixed number of chair seats, which I do not think is the case - despite our attempt to have a number of key companies represented. So, we should either state 'there will be 4 chairs', then this sentence about someone picking their replacement makes sense, or we should not talk about picking replacements at all and let things just happen via the normal nomination process. I don't think I was clear on this point in my previous comment.

MHBauer replied:
I don't think the "their replacement" implies a fixed number.

carolynvs replied:
The intent was to allow someone who is stepping down to participate in nominating and voting on people, i.e. they aren't lame ducks who as soon as they hint that they are on the way out, can't participate.

No number of replacements was meant to be implied.

How about "Chairs may decide to step down at any time. Before stepping down, the chair may continue to nominate new chairs and participate in the vote."

duglin replied:
"their replacement" implies a vacancy, which implies there's an assumed number of chairs rather then people become a chair based on something else.  Or put another way, why are we linking stepping down with adding new ones at all? To me they are totally independent actions.

arschles replied:
I like @carolynvs - I'm basing the following on that, making modifications to help signify that there are not a set number of chairs:

```
Chairs may decide to step down at any time. Before officially stepping down, the chair may decide to, but are not required to, nominate one or more new chairs and participate in votes for them. 
```

carolynvs replied:
@duglin What do you think of the new text that I suggested? It doesn't use the term replacement.

duglin replied:
@arschles its better but its still tortured. I'm not sure why there's a desire to link the two actions - perhaps I'm just missing something.  Why not just say `Chairs may decide to step down at any time. ` and that's it. Why do we need a 2nd sentence at all? the previous bullet talks about how to add chairs - what else is needed? What is the idea trying to be conveyed?  Why should one chair stepping down cause a new one to be added.... unless we're trying to find a replacement w/o calling it a "replacement" ;-)  If a chair that is stepping down really want to be involved in finding "another chair", then do the nominations before they step down.

arschles replied:
A few questions in here; I'll try to boil it down. The text acknowledges and encodes the fact that a chair can participate in finding someone to take their place, if they want someone to take their place. That's the idea to be conveyed here.

duglin replied:
> The text acknowledges and encodes the fact that a chair can participate in finding someone

The previous bullet already says that.

> to take their place

This is the rub. We didn't select this "other person" just because we think they should be a chair on their own, but instead its only because someone else is stepping down. This implies a link between the two actions - which is fine, if that's what we want, but then define it. Is it because there's a fixed number of seats? One from each key company? Does it imply we can't have more than one co-chair from a company? I'm not trying to be a pain, but there's an implied rule behind the proposed text and I'm looking for clarity on what it is.

Personally, I think the only thing we should state is that there can't be more than one co-chair from the same company. But just because someone from company X is stepping down doesn't mean we MUST find a replacement for them because there may not be someone else from that company who is active enough to justify it. And if we want to pick someone from another company, then we should have picked them regardless of the vacancy otherwise I would question if they were really worthy - meaning it shouldn't have been based on one person stepping down. And if it is due to someone stepping down, then that (again) implies some kind of fixed number of seats... so let's define that.

arschles replied:
Ok, I acknowledge that the previous bullets cover this bullet. I'm fine leaving it out. As you know, I agree with you that the number of chairs should not be limited at this point

carolynvs replied:
updated

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,131 @@
+    lazy two-thirds resolution amongst the chairs.
+
+- Emeritus Chairs ([Inspired by the Helm Project](http://technosophos.com/2018/01/11/introducing-helm-emeritus-core-maintainers.html))
+  - A chair who steps down may be given the title of Emeritus Chair. This title confers
```
"may".. does that mean its a vote or they get it automatically?

arschles replied:
let's make it their choice. could say "A chair who steps down may choose to take an Emeritus Chair title"

duglin replied:
LOL I agree with not forcing it on people

carolynvs replied:
good call, will update

MHBauer commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,131 @@
+
+## Roles
+
+- Maintainers
```
Since we don't use OWNERS file yet, we should write down who is Maintainer and who is a Chair, and either have that as part of this doc, or merged into the service-catalog repo beforehand.

carolynvs replied:
Seems easy enough to get the OWNERS file cleaned up first?

MHBauer replied:
Okay, I think OWNERS is probably fine as is. 

We should say MAINTAINERS == APPROVERS
and
we will move people to being REVIEWERS, and then maintainers/approvers.

MHBauer replied:
Chairs are listed in the Sig definition?

carolynvs replied:
Yes, the chairs are listed in `sigs.yaml`. I'm not sure I understand your comment about "maintainers and aprpovers and reviewers". Is that something that would happen after we move to PROW?

Otherwise, suggested verbiage would be greatly appreciated! 😀

MHBauer replied:
I think we can say for our first draft here, that "Maintainers is equivalent to the standard definition of Approvers".

MHBauer commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,131 @@
+    via lazy two-thirds resolution amongst the chairs, if they are unresponsive
+    for > 3 months or are not proactively working with other chairs to fulfill responsibilities.
+  - Chairs may propose changes to this charter at any time, to be accepted via
+    lazy two-thirds resolution amongst the chairs.
```
This seems very limiting unless we start adding many new chairs. I suggest we include maintainers, If maintainers are trusted enough to participate without higher level review, they should be trusted to belong in this process.

carolynvs replied:
Considering the entire charter thus far has been written by our SIG's maintainers, not chairs, I think this is a great suggestion. 👍 Fixing...

MHBauer commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,131 @@
+  - Maintainers may be nominated for removal from their position by a chair,
+    to accepted via lazy two-thirds resolution amongst the chairs.
+
+- Chairs
```
Chairs are listed in the sig definition https://github.com/kubernetes/community/tree/master/sig-service-catalog#chairs

MHBauer commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,133 @@
+  any topic, ultimately any changes made to the codebase must be
+  approved by the maintainers.
+- Pull requests are required to have at least 2 maintainers approve them using the `LGTM 1` and `LGTM 2` labels.
+- Pull requests that are labeled with `do not merge` or have an on-going
```
we should add the `/hold` prow plugin and consider using it for this marking.

carolynvs replied:
👍  My plan is to do a second pass at the charter after we move to PROW. I'll add an issue for that to the PROW milestone.

I don't want to have getting a charter in place mixed up with stuff that hasn't happened yet.

MHBauer replied:
good news! https://github.com/kubernetes/test-infra/commit/ad64db9bc945dce9bd618acd8a197d714c41de05

carolynvs replied:
HOT, let me play with it on an issue to make sure I got it right and I'll put it in.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,133 @@
+  - Maintainer is equivalent to the standard Kubernetes definition of Approver.
+  - Responsible for approving, and reviewing, pull requests.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers may be nominated by a chair, to accepted via lazy two-thirds
```
s/to/and/

duglin replied:
ahh, or perhaps you're missing the "be" after "to"?

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,133 @@
+  - New maintainers may be nominated by a chair, to accepted via lazy two-thirds
+    resolution amongst the chairs.
+  - Maintainers may be nominated for removal from their position by a chair,
+    to accepted via lazy two-thirds resolution amongst the chairs.
```
s/to/and/

duglin replied:
or add "be" after "to"

duglin commented:
overall LGTM with just 2 minor typos.  If you really wanna help my OCD, we could be more consistent on how we wrap lines :-)   I like at 80 cols

carolynvs commented:
@duglin Thanks, I've been staring at the carter so much that I forgot how to English. 😊 

Fixes and line-wrapification incoming

arschles commented:
/lgtm

carolynvs commented:
/assign @parispittman 

Can you help find the right people to lgtm and approve our charter now that we are all-on board?

fejta commented:
Please create an OWNERS file in your new sig-service-catalog directory so you don't have this problem next time

MHBauer commented:
@fejta we have something in current contents of owners file, but I don't know what it means or who is on these what-appear-to-be list accounts.
```
reviewers:
  - sig-service-catalog-leads
approvers:
  - sig-service-catalog-leads
labels:
- sig/service-catalog
```
https://github.com/kubernetes/community/blob/5422005a3f8901d6b2c5778f45b6666ffa3e27df/sig-service-catalog/OWNERS
Tracing through to
https://github.com/kubernetes/community/blob/5784e34efddbceb1cb5c7c77f78c090b1a5cadee/OWNERS_ALIASES#L93-L97
@arschles is on that list, so he should be able to have perms, I would think.

@carolynvs I didn't know about these owners files, is this something to list in the charter beyond the sig README? Are leads == chairs?

carolynvs commented:
I asked on Slack, and it turns out that team doens't exist in GitHub. 😂 

So I've opened a PR to manually add our chairs/leads until that's fixed:
https://github.com/kubernetes/community/pull/2286

carolynvs commented:
I've emailed the steering committee, thank you for the help in figuring out how to do this properly! 🙇‍♀️

carolynvs commented:
/hold cancel

pmorie commented:
/lgtm

duglin commented:
/lgtm
Thanks for all the work on this @carolynvs

cblecker commented:
/hold
This still needs steering committee approval.

carolynvs commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,158 @@
+  Simple majority (>50%) wins.
+
+### Assets
+- [Source Repository](https://github.com/kubernetes-incubator/service-catalog)
```
Good point! We recently went through an exercise with them to get our permissions fixed so that the current chairs have ownership. I'll add a sub-bullet explaining how it works and who owns what.

philips replied:
Perhaps this document should just list the single sub project and rely on the OWNERS file there:

https://github.com/kubernetes/community/blob/master/sigs.yaml#L1530

MHBauer replied:
That's now who has access to the github repo. That's only access to the source code.

philips commented:
Overall this look good to me. Just need to sort out the deal with the source repo naming vs calling it a subproject.

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+Service Catalog is a Kubernetes extension project that implements the [Open
+Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It allows
+application developers the ability to provision and consume cloud services
+natively from within Kubernetes.
```
👍

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+- Provide a UX for Kubernetes users that is consistent with both the OSB API
+  specification and traditional Kubernetes user interactions.
+- Align with the OSBAPI specification as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
```
Would you mind providing a link to your SIG's sigs.yaml file as well as a list of any code/binaries/services this SIG owns? We recently added an[ example SIG charter template](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md) if that's helpful to look at.

carolynvs replied:
Will do!

We do have a list further down of all the code/services/etc that we own. Do you need me to move it up into the Scope section?

https://github.com/carolynvs/community/blob/sig-service-catalog-charter/sig-service-catalog/charter.md#assets

michelleN replied:
yeah if you could make it look like that template, that would be awesome. thanks!!

carolynvs replied:
done!

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+## Roles
+
+- Maintainers
+  - Maintainer is equivalent to the standard Kubernetes definition of Approver.
```
if there is a link to what a Kubernetes `Approver` is, that would be super helpful.

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+    lazy two-thirds resolution amongst the chairs.
+
+- Chairs
+  - All maintainer’s roles.
```
`All maintainer's roles.` --> does that mean all maintainers are `Chairs`?

carolynvs replied:
Let me clarify that. The intent was to say that all Chairs are also expected to perform the role of maintainers as well.

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+    to a 2/3 majority vote (lazy two-thirds resolution).
+    This process is used for all decisions, such as changing chairs/maintainers
+    or modifying this charter.
+  - Chairs may nominate a new chair at any time, to be accepted via lazy
```
Is there a maximum number of `Chairs`?

carolynvs replied:
No, there is explicitly no set # of chairs. I'll add a note about that.

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+  - Chairs must remain active in the role and may be removed from the position
+    via lazy two-thirds resolution amongst the chairs, if they are unresponsive
+    for > 3 months or are not proactively working with other chairs to fulfill
+    responsibilities.
```
👍

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+    members can meet to discuss and solve problems for our SIG.
+
+### Project management
+- Milestones are defined by SIG maintainers.
```
Would be helpful to have a link to where you maintain milestones (non-blocking suggestion)

smarterclayton commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+  - Chairs may nominate a new chair at any time, to be accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Chairs may decide to step down at any time.
+  - Chairs must remain active in the role and may be removed from the position
```
Clarify if there is any other process for a chair to be removed other than by other chairs discussing.  I.e. can the maintainers decide chairs are out?

carolynvs replied:
@smarterclayton Do you have an example from another charter or suggestion about how maintainers could vote out an inactive chair? Currently we don't have a process for that though I could easily see it happening where multiple chairs are inactive and there's no defined way to fix that.

MHBauer replied:
I think we should have this, but the current charter was "how things are" not "how we wish they would be".

smarterclayton commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+  - Are a contact point for the Product Security Team to reach out to for
+    triaging and handling of incoming issues.
+  - Must be a maintainer.
+  - Must accept the Kubernetes [Embargo
```
Would suggest `accept and adhere to`

smarterclayton commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,159 @@
+  the `LGTM 1` and `LGTM 2` labels.
+- Pull requests that are labeled with `do-not-merge/hold` or have an on-going
+  discussion must not be merged until all concerns are addressed.
+- Disagreements are mainly resolved via consensus. In the event that a common
```
Consensus or lazy consensus?

carolynvs replied:
I'll update to specify lazy, as our goal is to use the same resolution process for as much as possible.

smarterclayton commented:
Left a few comments, also agree this looks good so far.

carolynvs commented:
I have addressed all comments except for https://github.com/kubernetes/community/pull/2233#discussion_r202416456 because I'm unclear on what an acceptable policy would be or if there is any precedent that we can reuse.

If we needed to add such a rule before this can be merged, then I would need to get approval from our chairs again as I don't expect whatever I come up with to be a slam dunk. 😉 

Would it be acceptable to follow up on this item after the charter is merged?

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,165 @@
+
+## Scope
+
+See the [service-catalog entry in sigs.yaml](https://github.com/kubernetes/community/blob/master/sigs.yaml#L1485).
```
either point to just the file (not the line number) since the line number can move (and already has), or just remove this sentence since I'm not sure that pointing to the sigs.yaml is really a better source of truth than what's in here.

philips replied:
Just link to the README here: https://github.com/kubernetes/community/tree/master/sig-service-catalog

That README is generated from sigs.yaml.

carolynvs replied:
Will do, thanks!

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,165 @@
+- Provide a UX for Kubernetes users that is consistent with both the OSB API
+  specification and traditional Kubernetes user interactions.
+- Align with the OSBAPI specification as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
```
Let's make "OSBAPI WG" a link to the OSPAPI WG.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,165 @@
+  reports, deep dives, etc.) should make their slides available for perusal and
+  feedback at least 2 week in advance.
+- [Working
+  groups](https://docs.google.com/document/d/1fghxARW-doHrNmBYCijhODsGoIFawpeU4X0end-XnUI/edit#)
```
Just to keep our doc-sprawl to a minimum, can we move this gdoc into a wiki page in our repo? Then its still easy to update but doesn't require a PR to do so.

carolynvs replied:
Yeah I was just linking to whatever doc was available but it would be better to have that in one place. I'll copy it into our wiki and update the charter to link to that.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,165 @@
+### Project management
+- [Milestones](https://github.com/kubernetes-incubator/service-catalog/milestones)
+  are defined by SIG maintainers.
+- Anyone is free to request for a discussion of the milestones/plans during
```
remove "for", it doesn't read quite right

carolynvs commented:
Per our agreement last week at our SIG's Face-to-Face, I have updated the charter so that any maintainer can nominate a new maintainer, not just a chair.

duglin commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,165 @@
+  - Those who attain this title are no longer expected to attend the weekly
+    meetings, share in the issue queue triage rotation, vote on policy or
+    architectural issues, or review pull requests.
+  - They are listed in our documentation as Emeritus Chairs, and we will
```
Link to the doc?

carolynvs replied:
Fixed

https://svc-cat.io/community/#leadership

duglin commented:
/LGTM

michelleN commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,166 @@
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes
+    leadership teams.
+  - All decisions amongst chairs are made using lazy consensus with a fallback
+    to a 2/3 majority vote (lazy two-thirds resolution).
```
(non-blocking): are there are a minimum number of chairs?

carolynvs replied:
We talked about it but we didn't feel the need to have a specific number. There is a line about that further down in the section

https://github.com/kubernetes/community/pull/2233/files#diff-d62d482f9a1782544ed6617b746e7ca2R92

`There are no set number of Chairs`

philips commented:
Thank you for your patience as we work to your SIG charter merged.

Through the process of reviewing the [first 11 charters][charters] the Steering Committee (SC) found a lot of copy/paste language caused by our poorly designed initial charter template. To fix this there is a new [SIG charter template][template] which focuses on the main things we want to see in charters: scope and responsibilities.

Please consider updating to this charter template and putting focus into the scope and responsibilities. SC members will focus more on a deep review of scope more than anything else and using this template will help with that focus.

[charters]: https://docs.google.com/spreadsheets/d/1i_8oM1KlmXo90zIIgNTdOg-M6xesTggFLnTkgmscMlI/edit#gid=0
[template]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md

carolynvs commented:
@philips I have reordered our sections (such as explaining our SIG's scope) so that it lines up with the template.

Since our charter was agreed upon by the SIG before the template was updated, and there were enough language differences, we decided to not rely on the template as a base charter. I kept all of our existing text and instead only call out major role differences, like that we don't have tech leads, but do have emeritus chairs.

In the future when the template is more settled (and we've had time to regroup as a SIG under our new charter), I hope to revisit and revise so that it can use more of the template.

philips commented on sig-service-catalog/charter.md:
```diff
@@ -0,0 +1,169 @@
+## Scope
+
+Service Catalog is a Kubernetes extension project that implements the [Open
+Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It allows
```
I disagree with this scope. People can consume cloud services natively in Kubernetes without service broker.

How about something like:

It enables application developers to provision cloud services from within Kubernetes and integrates configuration and discovery of those services into Kubernetes resources.

duglin replied:
I can live with that rewording

carolynvs replied:
Fixed

philips commented:
That is reasonable.

On Fri, Jul 20, 2018 at 1:33 PM Carolyn Van Slyck <notifications@github.com>
wrote:

> @philips <https://github.com/philips> I have reordered our sections (such
> as explaining our SIG's scope) so that it lines up with the template.
>
> Since our charter was agreed upon by the SIG before the template was
> updated, and there were enough language differences, we decided to not rely
> on the template as a base charter. I kept all of our existing text and
> instead only call out major role differences, like that we don't have tech
> leads, but do have emeritus chairs.
>
> In the future when the template is more settled (and we've had time to
> regroup as a SIG under our new charter), I hope to revisit and revise so
> that it can use more of the template.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/community/pull/2233#issuecomment-406719203>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACDCDqJD4juaZA4lhYpKmcDLSmXzcswks5uIj6XgaJpZM4Ugllx>
> .
>

carolynvs commented:
@philips I have updated the scope with your suggested wording. Let me know if there's anything else on my end holding up this PR. 😀

michelleN commented:
/lgtm

michelleN commented:
/hold

carolynvs commented:
@michelleN Is there anything else that needs to be done in order to merge this?

smarterclayton commented:
Looks fine to me

philips commented:
/hold cancel

both @smarterclayton and @michelleN from the Steering Committee agreed this looks OK.

There are future improvement to be made but merging this for now.

michelleN commented:
/approve
